io.github.tntwise.REAL-Video-Enhancer.yml

```diff
@@ -0,0 +1,27 @@
+app-id: io.github.tntwise.REAL-Video-Enhancer
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+command: REAL-Video-Enhancer
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=home
```

Comment:
this is a very broad filesystem permission.

Reply:
the default output folder is set to the home directory, and without these permissions video export will not work.